build/ci-build.yml

```diff
@@ -28,8 +28,6 @@ resources:
 
 variables:
   - group: 'Arcus Background Jobs - Integration Testing'
-  - group: 'Arcus Security - Integration Testing'
-  - group: 'Arcus Messaging - Integration Testing'
```

Comment:
Will step of re-using Azure resources from other projects, as this makes testing more complex. Seperate Azure resources is also easier to maintain.